src/current/_includes/releases/v22.2/v22.2.15.md

```diff
@@ -0,0 +1,75 @@
+- Fixed a bug where dependencies on [sequences](https://www.cockroachlabs.com/docs/v22.2/create-sequence) from tables would be reported with the wrong value for the `classid` column in the `pg_catalog.pg_depend` table. [#110211][#110211]
+- Fixed edge cases in [decimal](https://www.cockroachlabs.com/docs/v22.2/decimal) and [float](https://www.cockroachlabs.com/docs/v22.2/float) evaluation for division operators. `'NaN'::DECIMAL / 0` will now return `NaN` instead of a division-by-zero error, and `0 / 'inf'::DECIMAL` will return `0` instead of `0E-2019`. [#110298][#110298]
+- The `cockroach debug pebble` command now works correctly with [encrypted stores](https://www.cockroachlabs.com/docs/v22.2/security-reference/encryption#cockroachdb-self-hosted-clusters) which do not use the default `cockroach-data` path without having to also pass `--store`. [#110509][#110509]
+- When upgrading from previous releases, it was possible that ambiguous references to [sequences](https://www.cockroachlabs.com/docs/v22.2/create-sequence) would fail to be resolved by ID if they were in the same schema. Additionally, the logging is improved for ambiguous cases where the reference cannot be resolved. [#110721][#110721]
+- Fixed a bug where [`RESET `](https://www.cockroachlabs.com/docs/v22.2/row-level-ttl#reset-a-storage-parameter-to-its-default-value)[(`ttl_expire_after`)](https://www.cockroachlabs.com/docs/v22.2/row-level-ttl#using-ttl_expire_after) could incorrectly remove [`ttl_expiration_expression`](https://www.cockroachlabs.com/docs/v22.2/row-level-ttl#using-ttl_expiration_expression). [#110750][#110750]
+- Fixed a bug where [`format_type`](https://www.cockroachlabs.com/docs/v22.2/functions-and-operators#compatibility-functions) builtin did not honor `typemod` information for [array](https://www.cockroachlabs.com/docs/v22.2/array) types, leading to incorrect output. [#110939][#110939]
+- External connection URLs now accept the scheme `azure-blob` for connections to [Azure](https://www.cockroachlabs.com/docs/v22.2/use-cloud-storage#url-format) Blob Storage and the scheme `azure-kms` for connections to Azure KMS. For backward compatibility, schemes `azure` and `azure-storage` schemes continue to work for connections to Azure Blob Storage.
+- Fixed a bug where `indoption` inside [`pg_index`](https://www.cockroachlabs.com/docs/v22.2/pg-catalog#data-exposed-by-pg_catalog) was not properly encoded, causing it to be unable to be decoded as `int2vector`. [#111956][#111956]
+- Fixed a bug where the [optimizer](https://www.cockroachlabs.com/docs/v22.2/cost-based-optimizer) failed to honor the `statement_timeout` session setting when generating constrained index scans for queries with large `IN` lists or `= ANY` predicates on multiple index key columns, which may lead to an [out-of-memory crash (OOM)](https://www.cockroachlabs.com/docs/v22.2/cluster-setup-troubleshooting#out-of-memory-oom-crash) on the node. [#112075][#112075]
```

Comment:
Suggest making 'session setting' a link to https://www.cockroachlabs.com/docs/v22.2/session-variables

Reply:
Did not add link because https://www.cockroachlabs.com/docs/v22.2/session-variables does not exist.